### DISCUSSION TITLE
About cli

### DISCUSSION THREAD

**Discussion by JasonLi-cn** [databendlabs/databend#4596] [Q&A] [answered]
Why was the cli removed after version v0.6.90?

**Comment by zhang2014** [comment, marked as answer]
related: https://github.com/datafuselabs/databend/discussions/4423

**Reply by JasonLi-cn** [reply]
thx!

**Comment by bohutang** [comment]
It's easy to deploy databend by manual now:
https://databend.rs/doc/category/deploy

And all suggestions are welcomed.

**Reply by JasonLi-cn** [reply]
thx! I have successfully deployed it and am testing its performance.
I wonder how it compares to ClickHouse performance.

**Reply by bohutang** [reply]
> thx! I have successfully deployed it and am testing its performance. I wonder how it compares to ClickHouse performance.

Cool, maybe the ontime data(or yourself case) is a better way for the performance comparison, [Analyzing OnTime with Databend on Object Storage](https://databend.rs/doc/learn/analyze-ontime-with-databend-on-ec2-and-s3).
But make sure that Databend is develop for object storage not local disk.

**Reply by JasonLi-cn** [reply]
OK!
